requirements_all.txt

```diff
@@ -1313,7 +1313,7 @@ p1monitor==2.1.1
 paho-mqtt==1.6.1
 
 # homeassistant.components.panasonic_bluray
-panacotta==0.1
+panacotta==0.2
```

Comment:
This file is generated and should not be updated manually. Instead adjust the integration manifest and update our requirement files using the `gen_requirements_all` script. 

Just wondering how did you test this?

../Frenck

Reply:
Yes, you beat me to it. I have made the updates and submitted the PR. Tested locally by bumping versions manually, all looks good.

Reply:
1. Copy panasonic_bluray to custom_components and bump the requirements in manifest.json.  
2. Also to be safe, manually bump the requirements text files in /usr/src/homeassistant/ inside the container.
3. Reboot and test.

I wouldn't have submitted the PR if it wasn't working 😉

Reply:
Well, the thing is, you didn't test this PR :)

Additionally, you've checked these boxes even before you made the additional commit:

![CleanShot 2023-05-09 at 09 49 19@2x](https://user-images.githubusercontent.com/195327/237030313-34b36071-5d56-4023-9db3-dcee80efda18.png)

You checked, saying tests worked locally, and added tests to verify it's working. However, with a development environment, that is possible.

See our developer documentation on setting up a development environment here: <https://developers.home-assistant.io/docs/development_environment>

That, combined with the origin PR contents, and the `patch-1` branch name (indicating this PR was created from the GitHub web-UI), makes me hesitant to accept these types of, what is generally called, "drive-by" PRs.

../Frenck

Reply:
@kernelpanic85 I know you mean well; I was not questioning that ❤️ 

I was explaining why I triggered by this PR and why I hesitated. I did not mean to offend or doubt any of your goodness. However, as reviewers, we are also trying to guard and maintain quality.

> I'm not sure what else you're looking for on a simple dependency bump

Well, in all PRs, we expect the PR to be tested. Like the PR itself. Which wasn't the case. Additionally, you've checked boxes that turned out not to be true either.

So, don't get me wrong by saying this (it is not personal), but we look for correctness, quality, and reliability in reviews. The initial impression wasn't hitting those, which rings the alarm bells in my brain 🧠 

For the future, I would recommend setting up a dev environment. It is less scary than it seems, is pretty well documented, and doesn't need you to do weird things with custom integrations and stuff like that. You could even reach out in Discord for help in our developers channels, many would love to help.

Anyways, upstream looks good, the change looks fairly limited, let's move forward.

Nevertheless, thanks for the PR and the willingness to contribute 🤘 

../Frenck

Reply:
Thank you ❤️